### ISSUE TITLE
Order of validation report results

### ISSUE THREAD

**Issue by nicolas-raoul** [baturin/wikivoyage-listings#14]
The report results are grouped by article (which is great for efficient edition), but in addition I think the articles should be either:

- Organized on a map, so that people can process errors in places they are familiar with.
- Or randomized, so that people don't always process the same articles, and to prevent potential false-positives from making up most of the beginning of the list.

**Comment by baturin** [comment]
Issues are shuffled now by default, also there is a special button in the report.
As an example, check http://wvpoi.batalex.ru/download/listings/wikivoyage-listings-en-latest.validation-report.html

How do you see organizing that on map? By positional data of corresponding article?

**Comment by nicolas-raoul** [comment]
Yes, by positional data of corresponding article or POI latitude/longitude if it has some and they are valid.

**Comment by nicolas-raoul** [comment]
Here is the idea:
- A world map shows all cities that contain validation issues
- Clicking on one city shows all validation issues for that city (maybe in a kind of popup)
That makes it easy to find issues that are in regions I am familiar with.
